exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/spill/SpillSet.java

```diff
@@ -104,9 +107,17 @@
      * nodes provide insufficient local disk space)
      */
 
+    // The buffer size is calculated as LCM of the Hadoop internal checksum buffer (9 * checksum length), where
```

Comment:
@paul-rogers Changed `TRANSFER_SIZE` to 72K. It is calculated as LCM between Hadoop internal checksum buffer and MapR FS page size to make writes aligned on internal buffer boundaries.